ProcessHacker/proctree.c

```diff
@@ -1380,57 +1380,54 @@ static VOID PhpUpdateProcessNodeDpiAwareness(
             return;
     }
 
-    if (!(ProcessNode->ValidMask & PHPN_DPIAWARENESS))
```

Comment:
This doesn't actually remove the flags.

Reply:
I'm not sure if I follow you. Do you want PHPN_DPIAWARENESS to be removed completely? I wasn't sure if it was being used somewhere else.